src/samples/README.md

```diff
@@ -0,0 +1,143 @@
+# Google Cloud Platform C++ Samples
+
+A collection of samples that demonstrate how to call some
+Google Cloud services from C++.
+
+## Prerequisites
+
+See the instructions in the root [README](../../README.md) to build.
+
+Before calling a Google API, you must create a Google Cloud project and enable
+billing.
+
+1.  **Create a project in the Google Cloud Platform Console**.
+    If you haven't already created a project, create one now. Projects enable
+    you to manage all Google Cloud Platform resources for your app, including
+    deployment, access control, billing, and services.
+    1.  Open the [Cloud Platform Console](https://console.cloud.google.com/).
+    2.  In the drop-down menu at the top, select **Create a project**.
+    3.  Click **Show advanced options**. Under App Engine location, select a
+        United States location.
+    4.  Give your project a name.
+    5.  Make a note of the project ID, which might be different from the project
+        name. The project ID is used in commands and in configurations.
+
+2.  **Enable billing for your project**.
+    If you haven't already enabled billing for your project,
+    [enable billing now](https://console.cloud.google.com/project/_/settings).
+    Enabling billing allows the application to consume billable resources such
+    as running instances and storing data.
+
+## Running the Google Cloud Storage Sample
+
+The Cloud Storage sample uses [Service Account](
+    https://developers.google.com/identity/protocols/OAuth2ServiceAccount)
+credentials to authenticate with the Cloud Storage API.
+
+1.  **Enable APIs.**
+
+    [Click here](
+        https://console.cloud.google.com/flows/enableapi?apiid=storage-component.googleapis.com&showconfirmation=true)
+    to enable the Google Cloud Storage API for your project.
+
+2.  **Download credentials**.
+
+    1.  [Visit the API Manager Dashboard](https://console.cloud.google.com/apis/dashboard)
+        and click **Credentials**.  
+    2.  Click **Create credentials** and choose **Service Account key**.
+    3.  Under Service account, choose **Compute Engine default service
+        account**, and leave **JSON** selected under Key Type.  Click
+        **Create**.  A .json file will be downloaded to your computer.
+
+3.  **Run storage_sample**.
+
+    Pass the path of the .json file you downloaded to the storage_sample:
+
+        $ bin/storage_sample ~/Downloads/your-project-id-1cf578086929.json
```

Comment:
Change **~\/Downloads\/your-project-id-1cf578086929\.json** to **path/to/service_account.json**